src/js/pagination/paging-collection.js

```diff
@@ -83,7 +83,15 @@
              * backbone.paginator.
              */
             parseState: function (response, queryParams, state, options) {
-                return {totalRecords: response[0].count, totalPages: response[0].num_pages};
+                state.totalRecords = response[0].count;
```

Comment:
"This method MUST return a new state object instead of directly modifying the PageableCollection#state object. The behavior of directly modifying PageableCollection#state is undefined."

@dan-f Well what this statement tries to convey is that we must return a state object instead of just  modifying the state and returning nothing in which case state is undefined. So this state that is being passed to this function is being passed a copy not the actual state object. So it's safe to modify and return the state. This is not the collection maintained state.